### PR TITLE
[Feature] CON 81 - Enable text wrap passages

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to this issue**

https://oat-sa.atlassian.net/browse/CON-81

**Other related PRs**

- https://github.com/oat-sa/extension-tao-itemqti/pull/1580
- https://github.com/oat-sa/extension-tao-mediamanager/pull/240

**Story**

As the author, I want to be able to wrap my text around pictures, so that I can produce passages that look like real-life media content.

Currently, I am only able to anchor images AS a character. In order for the text to be able to flow around it, I will need to be able to anchor it TO a character and specify how text is supposed to wrap around it

Required wrap modes:

- Float off
- Float right
- Float left

**Acceptance Criteria**

- Anchoring an image as a character should still be possible
- Text wrapping should not be available when the image is anchored as a character
- Images maintain position within their paragraph when text is added before or after
- Responsive images are still supported if the image is anchored to a character
- Exported items will contain the relevant CSS to ensure the rendering will be correct in any environment
- Exporting and reimporting an item will maintain the anchoring and wrapping

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "@oat-sa/tao-core-libs": "0.4.3",
         "@oat-sa/tao-core-sdk": "^1.12.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.22.16",
+        "@oat-sa/tao-core-ui": "git://github.com/oat-sa/tao-core-ui-fe#feature/CON-81/enable-text-wrap-passages",
```
Comment (404passionFruit) [inline views/package.json:16]: Just a comment to not forget change this to a proper version

File: views/package-lock.json [hunk 2]
```diff
@@ -39,9 +39,8 @@
             "integrity": "sha512-S6/2ol3NpT99su9ZlZgoVvZHKW+yRVro562Ab1pizyCwv2YfO1r2FbpEDTS/cgfqI2JvgMtuP1V1SjR3T8Avrg=="
         },
         "@oat-sa/tao-core-ui": {
-            "version": "1.22.16",
-            "resolved": "https://registry.npmjs.org/@oat-sa/tao-core-ui/-/tao-core-ui-1.22.16.tgz",
-            "integrity": "sha512-Pn3/G80bWj14O6bDBoQnzStLfwQyx2WC2zoS79njLJQlEhAUykduKBGEIQlf3jvOtVtG5QbrVF4aUriYtBv0Sw=="
+            "version": "git://github.com/oat-sa/tao-core-ui-fe.git#a37ec8686a7131bfd8a3e0fbd42c63a43e61c2f9",
```
Comment (404passionFruit) [inline views/package-lock.json:42]: Just a comment to not forget change this to a proper version

**Comment by 404passionFruit** [comment]
Converted to draft as there is an alternative solution https://github.com/oat-sa/tao-core/pull/2936

**Comment by habemuscode** [comment]
Deleted in favor of https://github.com/oat-sa/tao-core/pull/2936

**Pull request closed without merging**